src/lipalightninglib.udl

```diff
@@ -40,6 +40,11 @@ interface LightningNode {
     [Throws=LnError]
     LspFee query_lsp_fee();
 
+    // Limits in the amount sats a payment is allowed to transfer. Currently there are only limits for receiving payments.
+    // The limits (partly) depend on the channel situation of the node, so it should be called again every time the user is about to receive a payment.
```

Comment:
```suggestion
    // The limits (partly) depend on the channel situation of the node, so it should be called every time the user is about to issue an invoice.
```

Reply:
That change doesn't cover spontaneous payments anymore .. (I'm particularly thinking of the use case when a payment has to be pulled down via LNURLw)

Reply:
An invoice still have to be issued when using LNURLw. But I agree that the user of this library might not realize it. I proposed the change because the original version makes it seem like this should be used right before the payment was received and not before the invoice was issued.

Reply:
From a UX point of view there is no creation of any invoice.